### PR TITLE
Fixes for documentation of installation on GKE

### PR DESCRIPTION
**Pull request by ivanmalamen**
**Description**
Fixes for documentation of installation on GKE
Changes proposed in this pull request:

Line 75: Removed sudo, as key and cert are not readable by further EXPORT commands - they are created with "root" owner
Line 107: 1.10.7-gke.6 changed to latest, as hardcoded version becomes obsolete as time goes on
Line 123, 124: Download latest config file
Line 129, 130: Additional sed, as $TLS_KEY and $TLS_CERT contains spaces, that need to be escaped. Otherwise you get "sed: -e expression kyma-project#1, char 90: unterminated `s' command" error

**Related issue(s)**

### CODE REVIEW

File: docs/kyma/docs/032-gs-gke-installation.md [hunk 1]
```diff
@@ -119,11 +118,17 @@ Delegate the management of your domain to Google Cloud DNS. Follow these steps:
...
+1. Download the `kyma-config-cluster` file bundled with the latest Kyma [release](https://github.com/kyma-project/kyma/releases/). Run:
+   ```
+   LATEST=$(curl https://github.com/kyma-project/kyma/releases/latest -I|grep Location:| rev | cut -d'/' -f1 | rev|tr -d '\r')
+   wget https://github.com/kyma-project/kyma/releases/download/${LATEST}/kyma-config-cluster.yaml
+   ```
 
 2. Update the file with the values from your environment variables. Run:
     ```
-    cat kyma-config-cluster.yaml | sed -e "s/__DOMAIN__/$DOMAIN/g" |sed -e "s/__TLS_CERT__/$TLS_CERT/g" | sed -e "s/__TLS_KEY__/$TLS_KEY/g"|sed -e "s/__.*__//g"  >my-kyma.yaml
+    ESC_TLS_CERT=$(echo $TLS_CERT|sed 's/ /\\ /g')
```
Comment (pbochynski) [inline docs/kyma/docs/032-gs-gke-installation.md]: Please move the escaping to the section where TLS_CERT is set.

**Comment by pbochynski** [comment]
Thank you for your contribution. There is only one comment to fix before it can be merged.

**Comment by tomekpapiernik** [comment]
@ivanmalamen Thank you for the contribution. Please act on Piotr's comment. I'll approve the PR when you apply the changes.

**Comment by ivanmalamen** [comment]
Fixed. Could you please merge?

**Pull request merged**